config/impl/src/main/java/org/jboss/windup/config/loader/LabelLoaderImpl.java

```diff
@@ -0,0 +1,112 @@
+package org.jboss.windup.config.loader;
+
+import org.jboss.forge.furnace.proxy.Proxies;
+import org.jboss.forge.furnace.services.Imported;
+import org.jboss.windup.config.LabelProvider;
+import org.jboss.windup.config.metadata.Label;
+import org.jboss.windup.config.metadata.LabelProviderRegistry;
+import org.jboss.windup.util.ServiceLogger;
+import org.jboss.windup.util.exception.WindupException;
+
+import javax.inject.Inject;
+import java.util.*;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class LabelLoaderImpl implements LabelLoader
```

Comment:
Please add comments to describe the class.

Reply:
done